lib/composables/Table.ts

```diff
@@ -21,7 +21,7 @@ export interface Table<
   reset?: MaybeRef<boolean | undefined>
   borderless?: MaybeRef<boolean>
   loading?: MaybeRef<boolean | undefined>
-  virtualScroll?: MaybeRef<boolean | undefined>
+  rowSize?: MaybeRef<number | undefined>
```

Comment:
this should include border width as well, so if the row height is 40px and there is 1px border, it should be 41px